package.json

```diff
@@ -19,8 +19,8 @@
       "ccdUrl": "http://ccd-data-store-api-demo.service.core-compute-demo.internal"
     },
     "aat": {
-      "cosUrl": "http://prl-cos-aat.service.core-compute-aat.internal",
-      "ccdUrl": "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
+       "cosUrl": "http://prl-cos-pr-384.service.core-compute-preview.internal/",
+       "ccdUrl": "http://ccd-data-store-api-prl-ccd-definitions-pr-409.service.core-compute-preview.internal"
```

Comment:
Remove this change and it should be all good.

Reply:
remove this after testing